<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SslTrustEditor</class>
 <widget class="QWidget" name="SslTrustEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_23">
     <property name="text">
      <string>Trust Level</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="trust_level"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_24">
     <property name="text">
      <string>Certificate Authorities</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QRadioButton" name="trust_enable_ca">
       <property name="text">
        <string>Use local certificate authorities</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="trust_disable__ca">
       <property name="text">
        <string>Don't use local certificate authorities</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_25">
     <property name="text">
      <string>Trusted Hosts</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTableView" name="trusted_hosts">
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="cornerButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QToolButton" name="trust_revoke_selected">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Revoke trust</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
